CONTRIBUTING.md

```diff
@@ -2,6 +2,8 @@
 
 The issues are used to track both bugs filed by users and specific work items for developers. Try to file one issue per problem observed. Please specify a valid title (e.g. "Glyph spacing is incorrect" instead of "PDF.js does not work") and provide more details about the issue: link to the PDF, location in the PDF, screenshot, browser version, operating system, PDF.js version and JavaScript console warning/error messages. Issues that do not have enough details provided will be closed as invalid/incomplete.
 
+We understand that many PDFs contain sensitive information, however having a PDF to reproduce and test is essential to resolving the issue and building regression testing. If possible try creating a reduced example exhibiting the problem but not containing sensitive data. Also small PDFs are best suited for our regression testing. If an important issue only shows on sensitive PDFs, contributors might be willing to accept these PDFs via a secure exchange. If a PDF cannot be provided, the issue is most likely to be closed.
```

Comment:
Personally, I'd even consider going as far as saying that if there is *no* way for the reporter to provide a test-case, then there's no point in opening as issue.
But that is perhaps a bit too undiplomatic :-)